modules/installation-creating-gcp-security.adoc

```diff
@@ -48,40 +58,97 @@ resources:
 EOF
 ----
-<1> `infra_id` is the `INFRA_ID` infrastructure name from the extraction step.
-<2> `region` is the region to deploy the cluster into, for example `us-east1`.
-<3> `cluster_network` is the `selfLink` URL to the cluster network.
-<4> `network_cidr` is the CIDR of the VPC network, for example `10.0.0.0/16`.
-<5> `master_nat_ip` is the IP address of the master NAT, for example `34.94.100.1`.
-<6> `worker_nat_ip` is the IP address of the worker NAT, for example `34.94.200.1`.
+<1> `allowed_external_cidr` is the CIDR range that can access the cluster API and SSH to the bootstrap host. For an internal cluster, set this value to `${NETWORK_CIDR}`.
+<2> `infra_id` is the `INFRA_ID` infrastructure name from the extraction step.
+<3> `region` is the region to deploy the cluster into, for example `us-east1`.
+<4> `cluster_network` is the `selfLink` URL to the cluster network.
+<5> `network_cidr` is the CIDR of the VPC network, for example `10.0.0.0/16`.
+<6> `master_nat_ip` is the IP address of the master NAT, for example `34.94.100.1`.
+<7> `worker_nat_ip` is the IP address of the worker NAT, for example `34.94.200.1`.
 
 . Create the deployment by using the `gcloud` CLI:
 +
 ----
 $ gcloud deployment-manager deployments create ${INFRA_ID}-security --config 03_security.yaml
 ----
 
+. Export the variable for the master service account:
++
+----
+$ export MASTER_SA=${INFRA_ID}-m@${PROJECT_NAME}.iam.gserviceaccount.com
+----
+
+. Export the variable for the master service account:
++
+----
+$ export WORKER_SA=${INFRA_ID}-w@${PROJECT_NAME}.iam.gserviceaccount.com
+----
+
+ifdef::shared-vpc[]
+. Assign the permissions that the installation program requires to the service accounts for the subnets that host the control plane and compute subnets:
+
+.. Export the variable for the name of the account that hosts the shared VPC:
++
+----
+$ export HOST_PROJECT_ACCOUNT=<host_project_account>
+----
+
+.. Export the variable for the name of the project that hosts the shared VPC:
++
+----
+$ export HOST_PROJECT=<host_project>
+----
+
+.. Grant the `networkViewer` role to the master service account:
++
+----
+$ gcloud --account=${HOST_PROJECT_ACCOUNT} --project=${HOST_PROJECT} projects add-iam-policy-binding ${HOST_PROJECT} --member "serviceAccount:${MASTER_SA}" --role "roles/compute.networkViewer"
+----
+
+.. Grant the `networkUser` role to the master service account for the control plane subnet:
++
+----
+$ gcloud --account=${HOST_PROJECT_ACCOUNT} --project=${HOST_PROJECT} compute networks subnets add-iam-policy-binding "${HOST_PROJECT_CONTROL_SUBNET}" --member "serviceAccount:${MASTER_SA}" --role "roles/compute.networkUser"
+----
+
+.. Grant the `networkUser` role to the worker service account for the control plane subnet:
++
+----
+$ gcloud --account=${HOST_PROJECT_ACCOUNT} --project=${HOST_PROJECT} compute networks subnets add-iam-policy-binding "${HOST_PROJECT_CONTROL_SUBNET}" --member "serviceAccount:${WORKER_SA}" --role "roles/compute.networkUser"
```

Comment:
Need to specify region like this:

    gcloud --account=${HOST_PROJECT_ACCOUNT} --project=${HOST_PROJECT} compute networks subnets add-iam-policy-binding "${HOST_PROJECT_CONTROL_SUBNET}" --member "serviceAccount:${WORKER_SA}" --role "roles/compute.networkUser" --region ${REGION}